homeassistant/components/zha/number.py

```diff
@@ -947,3 +949,25 @@ class AqaraThermostatAwayTemp(ZHANumberConfigurationEntity):
     _attr_mode: NumberMode = NumberMode.SLIDER
     _attr_native_unit_of_measurement: str = UnitOfTemperature.CELSIUS
     _attr_icon: str = ICONS[0]
+
+
+@MULTI_MATCH(
+    cluster_handler_names=CLUSTER_HANDLER_THERMOSTAT,
+    stop_on_match_group=CLUSTER_HANDLER_THERMOSTAT,
```

Comment:
stop_on_match_group is meant to prevent different implementations for the same functionality from being matched. Do you need this for this entity?

Reply:
addressed with
https://github.com/home-assistant/core/pull/105765/commits/3435b63b82ce963cfc9c606cb9aadf91fe04a695